docs/StepbyStepManualDeploymentGuideScalarDLOnAzure.md

```diff
@@ -0,0 +1,237 @@
+# Example steps to Deploy Scalar DL on Azure
```

Comment:
The filename should probably be like `StepByStep...`.

Reply:
Thank you, updated file name in 8f9b0ff